Bootstrap navigation bar

mtuchi opened the pull request:
i added bootstrap.js in order to make a navbar responsive

kynan commented:
I'm not entirely sure what you did there, but this is most definitely not what you intended I think. There are lots of conflict markers and files committed you certainly didn't mean to commit.

kynan commented:
So I think I could somewhat untangle what is going on here: It seems you created commit f7dcd10 on top of master (which is probably the one you wanted for this pr - and only that one!). After that however it seems you merged some other branch with the code of all your previous pull requests and made a total mess. I assume you didn't really want that. However commit f7dcd10 also contains a lot of random changes which I think are unintentional. Not sure what to suggest to clean this up. What did you actually want to do?

mtuchi commented:
I wanted to update my source for the  navigation bar only did i commit in a
wrong file again? Because the only file i have worked with is index.html
main.css main.html and i added a bootstrap.js link from their website
I figured out that this could solve number of issue for a mobile friendly
environment
However i intend to write a media queries on the main.html
The filter panel is taking alot of space in  small device
On 14 Jan 2015 22:43, "Florian Rathgeber" <notifications@github.com> wrote:

> So I think I could somewhat untangle what is going on here: It seems you
> created commit f7dcd10
> <https://github.com/taarifa/TaarifaWaterpoints/commit/f7dcd109150d2951d5b9de6bcab9948078917f2a>
> on top of master (which is probably the one you wanted for this pr - and
> only that one!). After that however it seems you merged some other branch
> with the code of all your previous pull requests and made a total mess. I
> assume you didn't really want that. However commit f7dcd10
> <https://github.com/taarifa/TaarifaWaterpoints/commit/f7dcd109150d2951d5b9de6bcab9948078917f2a>
> also contains a lot of random changes which I think are unintentional. Not
> sure what to suggest to clean this up. What did you actually want to do?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/taarifa/TaarifaWaterpoints/pull/182#issuecomment-69978187>
> .
>

mtuchi commented:
So what i did before i commit i updated my local repository by git pull
command the i add my changes
git add .
Then git commit -m
Then i check the status
git status
If everything is okay i push my sourcecode to my github account
On 15 Jan 2015 08:36, "mgombere" <emmanuelevance1@gmail.com> wrote:

> I wanted to update my source for the  navigation bar only did i commit in
> a wrong file again? Because the only file i have worked with is index.html
> main.css main.html and i added a bootstrap.js link from their website
> I figured out that this could solve number of issue for a mobile friendly
> environment
> However i intend to write a media queries on the main.html
> The filter panel is taking alot of space in  small device
> On 14 Jan 2015 22:43, "Florian Rathgeber" <notifications@github.com>
> wrote:
>
>> So I think I could somewhat untangle what is going on here: It seems you
>> created commit f7dcd10
>> <https://github.com/taarifa/TaarifaWaterpoints/commit/f7dcd109150d2951d5b9de6bcab9948078917f2a>
>> on top of master (which is probably the one you wanted for this pr - and
>> only that one!). After that however it seems you merged some other branch
>> with the code of all your previous pull requests and made a total mess. I
>> assume you didn't really want that. However commit f7dcd10
>> <https://github.com/taarifa/TaarifaWaterpoints/commit/f7dcd109150d2951d5b9de6bcab9948078917f2a>
>> also contains a lot of random changes which I think are unintentional. Not
>> sure what to suggest to clean this up. What did you actually want to do?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/taarifa/TaarifaWaterpoints/pull/182#issuecomment-69978187>
>> .
>>
>

kynan commented:
Better only `git add` the file you actually changed. Doing `git add .` means you keep adding files to the repository we don't want to have tracked (log files etc.). Also, check the status *before* you commit to make sure you're only commit the changes you intend.

kynan commented:
The problem with you changes in  f7dcd10 is also that you seem to have lost all leading white space, not sure how that happened. It's really important that you don't change any unrelated files or unrelated sections of the file you're modifying since that makes it very hard to see what you're actually trying to do.

kynan commented:
This would require a lot of cleaning and I'm not sure what this patch is trying to achieve. @dgorissen?

dgorissen commented:
me neither, would shelve it
